lang/cs/Org.Apache.REEF.Client.Tests/packages.config

```diff
@@ -18,7 +18,7 @@ specific language governing permissions and limitations
 under the License.
 -->
 <packages>
-  <package id="Newtonsoft.Json" version="6.0.8" targetFramework="net45" />
+  <package id="Newtonsoft.Json" version="7.0.1" targetFramework="net45" />
```

Comment:
There might be a way, if we declare out packages in solution-level packages.config (I think that's the one in lang\cs\.nuget) instead of per-project packages.config. But packages declared there behave in a different way, sometimes they require restoring them manually instead of automatically etc.

I agree that it's a good thing to have, that would probably allow us to avoid REEF-1015 and just be convenent. But this is something I need to explore more, so I'd rather decouple it from this PR. Let's file a JIRA for this